Client/src/components/Shared/Navbar/Navbar.jsx

```
@@ -3,133 +1,147 @@
+                        <p className="text-xs text-gray-400 truncate">{user.email}</p>
                       </div>
+                      <Link to="/profile" className="block px-4 py-2 text-sm text-gray-300 hover:bg-gray-700 hover:text-white transition-colors">
+                        Your Profile
+                      </Link>
+                      <Link to="/dashboard" className="block px-4 py-2 text-sm text-gray-300 hover:bg-gray-700 hover:text-white transition-colors">
+                        Dashboard
+                      </Link>
+                      <Link to="/settings" className="block px-4 py-2 text-sm text-gray-300 hover:bg-gray-700 hover:text-white transition-colors">
+                        Settings
+                      </Link>
+                      <div className="border-t border-gray-700 my-1"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)